src/ripple/app/tx/impl/SignerEntries.h

```diff
@@ -25,50 +25,42 @@
 #include <ripple/protocol/STTx.h>           // STTx::maxMultiSigners
 #include <ripple/protocol/TER.h>            // temMALFORMED
 #include <ripple/protocol/UintTypes.h>      // AccountID
+#include <optional>
 
 namespace ripple {
 
 // Forward declarations
 class STObject;
 
-// Support for SignerEntries that is needed by a few Transactors
-class SignerEntries
+/** An individual entry in a signer list */
+struct SignerEntry
```

Comment:
Having this structure makes code self-documenting. I'm curious if anyone thinks that `std::pair<AccountID, std::uint16_t>` would be better; it gives us the two comparison operators we want for "free" (not that the cost we're paying now is a lot).